### PR TITLE
Cleanup MatrixInfo (remove Jython hack; black like formatting)

### PR DESCRIPTION
**Pull request by peterjc**
This pull request addresses issue #2008 by disabling black on this file, and is part of the general Python 3 cleanup by dropping a Jython JVM workaround.

Black formatting is not appropriate for these dictionaries.

The line breaking has not changed, so indentation and quote character aside, they should be identical to the original checking bar a bug fix for the ``grant`` matrix and renaming of the ``structure`` matrix (originally just ``str`` which clashes):

```
$ unify --in-place Bio/SubsMat/MatrixInfo.py # revert to single quotes
$ git diff -w f223c67bc8a02289d1dcaa1d2b3d7edd48a7742b Bio/SubsMat/MatrixInfo.py
...
```

- [X] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [X] I have read the ``CONTRIBUTING.rst`` file, have run ``flake8`` locally, and
understand that AppVeyor and TravisCI will be used to confirm the Biopython unit
tests and style checks pass with these changes.

- [X] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

### CODE REVIEW

**Comment by peterjc** [comment]
@cbrueffer you've worked on this file from a style point of view - can you review the changes?

The JVM hack dates back to 2b2910fa0fb0a57f8a9656092fbb4653ce048b28

**Comment by peterjc** [comment]
@MarkusPiotrowski could you review this?

**Comment by peterjc** [comment]
@svalqui can you look at this? Please do double check the data values have not changed - for example with the diff I used in the pull request description.

**Comment by peterjc** [comment]
How about you @mdehoon - could you review this?

**Comment by mdehoon** [comment]
@peterjc  MatrixInfo is obsolete code. You may as well remove it and point users to `Bio/Align/substitution_matrices`.

**Comment by peterjc** [comment]
In that case we should label is as obsolete, add the missing matrixes like Grant to ``Bio.Align.substitution_matrices`` and then look to deprecate ``Bio,SubsMat,MatrixInfo``.

May I assume you had not objections to the changes here? I'd like to do this anyway as we are now very close to getting all of ``Bio/`` black formatted.

**Comment by mdehoon** [comment]
@peterjc  I have no objections.

**Comment by peterjc** [comment]
Thanks - applying now...

File: Bio/SubsMat/MatrixInfo.py [hunk 1]
```diff
@@ -35,2888 +37,2691 @@
```
Comment (mdehoon) [inline Bio/SubsMat/MatrixInfo.py]: The comment above can now be removed.

Comment (peterjc) [reply Bio/SubsMat/MatrixInfo.py]: Good catch, yes - removing the ``_temp`` functions was the main change in the pull request.

**Pull request merged**